staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -270,3 +270,8 @@ func (list ErrorList) Filter(fns ...utilerrors.Matcher) ErrorList {
 	// FilterOut takes an Aggregate and returns an Aggregate
 	return fromAggregate(err.(utilerrors.Aggregate))
 }
+
+// Implementing Is error method
+func (m Error) Is(target error) bool {
+	return target.Error() == m.Error()
```

Comment:
This should check the type of the error and compare the actual fields in `target`. Please have a look at https://blog.golang.org/go1.13-errors. It has an excellent example of Is implementation.

Please implement it on the pointer receiver similarly to other methods for this type.

The comment should also reflect what the method is doing using Godoc convention: https://blog.golang.org/godoc